packages/client-adapters/src/adapters/tufts/engine/sedra.js

```diff
@@ -11,7 +11,7 @@ data.setMeaningParser(function (meaning, targetWord) {
   return new Models.Definition(meaningText, lang, 'text/plain', targetWord)
 })
 
-data.setPropertyParser(function (propertyName, propertyValue) {
+data.setPropertyParser(function (propertyName, propertyValue, inputElem) {

```

Comment:
Bridget, I am sorry, may be I did miss something.
I see how you added inputElem to arguments of parserProperty - but I don't see any usage inside functions of this argument.
May be it is not needed?

Reply:
I changed the function signature because I needed the input element in the morpheusgrc engine to check if the property was in the hdwd